source/modules/configuration/notifications/email_notifications.rst

```diff
@@ -11,19 +11,36 @@ Global Configuration
 --------------------
 
 The global configuration allows you to set the following options:
-- Administrator email - GLPI Administrator email. This can be used as a special recipient in notifications.
-- Administrator name - The display name to use for the Administrator email address.
-- From email - The email address to use in the From field for emails sent by GLPI.
-- From name - The display name to use for the From email address.
-- Reply-to address - The email address to use when user's reply to the email notifications.
-- Reply-to name - The display name to use for the Reply-to email address.
-- No-Reply address - The email address to use for emails that shouldn't be responded to.
-- No-Reply name - The display name to use for the No-Reply email address.
-- Add documents into ticket notifications - If enabled, documents attached to a ticket will be added to related notifications as links. The links will use the GLPI URL specified in the general configuration.
-- Email signature - Text added at the end of every notification.
-- Way of sending emails - Method to send emails (PHP, SMTP, SMTP+SSL, SMTP+TLS).
-- Max delivery retries - Number times GLPI will try to send a notification.
-- Try to deliver again in (minutes) - Time between notification delivery attempts.
+
++------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
I'm not sure table is the better option for display; it currently adds an horizontal scroll :/

Reply:
Hmmm...

I'm working on it. There's something called [Line blocs](https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#line-blocks) that I think can be useful to solve both problems:
 - Horizontal scrolling (adding more line breaks)
 - The need to declare the `br` snippet. That can be useful, but I think it's better to avoid it. 

Line blocks even works on tables (I was not sure about this). The code:
```
+------------------------------------------------------+------------------------------------------------------------------------------------------------+
|                           **Option**                 |                                  **Meaning**                                                   |
+------------------------------------------------------+------------------------------------------------------------------------------------------------+
| `Administrator email`                                | | GLPI Administrator email.                                                                    |
|                                                      | | This can be used as a special recipient in notifications                                     |
+------------------------------------------------------+------------------------------------------------------------------------------------------------+
```
Generates:
![image](https://github.com/glpi-project/doc/assets/44492590/6c4565a2-8d83-4063-8640-3f4196e8e407)

I'm working on a CSS custom rule to avoid the added below line space. Editing the following rule (only for the table(s), of course) looks to solve the problem:
![image](https://github.com/glpi-project/doc/assets/44492590/31a5c94b-44fe-489a-aab9-8876bc0f616a)

Waiting for feedback on how to continue. Thanks!

Reply:
> [...]
> Line blocks even works on tables (I was not sure about this). The code:

Sounds good to me :) I also agree it would be better not to declare `br` at all.

> I'm working on a CSS custom rule to avoid the added below line space. Editing the following rule (only for the table(s), of course) looks to solve the problem:
> [...]

:+1:

Reply:
Updated. Waiting for your review :)

Reply:
This is way better, thank you :)